src/javascript/crypto/e2e/extension/ui/prompt/prompt.js

```diff
@@ -365,27 +216,20 @@ ui.Prompt.prototype.renderMenu_ = function(elem, blob, opt_showReduced) {
  * keyring.
  * @param {Element} elem The element into which the UI elements are to be
  *     rendered.
- * @param {?messages.BridgeMessageRequest} contentBlob The content that the user
- *     has selected.
  * @private
  */
-ui.Prompt.prototype.renderKeyringPassphrase_ = function(elem, contentBlob) {
+ui.Prompt.prototype.renderKeyringPassphrase_ = function(elem) {
   var dialog = new dialogs.Generic(
       '',
       goog.bind(function(passphrase) {
         try {
           // Correct passphrase entered.
           this.pgpLauncher_.start(passphrase);
-          if (contentBlob && contentBlob.action) {
-            if (contentBlob.action == constants.Actions.GET_PASSPHRASE) {
-              contentBlob.action = undefined; // Fall into the default action.
-            }
-            this.processSelectedContent_(contentBlob, contentBlob.action);
-          } else {
-            this.close();
-          }
+          this.processSelectedContent_();
```

Comment:
should be `this.close()` unless it's intended to change the original behavior (i.e., closing the dialog after correct passphrase is entered).